tests/test_mapdl.py

```diff
@@ -992,4 +992,29 @@ def test_inquire(mapdl):
     # **Returning Information About a File to a Parameter**
     jobname = mapdl.inquire("", 'jobname')
     assert float(mapdl.inquire("", 'exist', jobname + '.lock')) in [0, 1]
-    assert float(mapdl.inquire("", 'exist', jobname , 'lock')) in [0, 1]
+    assert float(mapdl.inquire("", 'exist', jobname, 'lock')) in [0, 1]
+
+
+@pytest.mark.parametrize("option2", ['expdata.dat', 'expdata', 'expdata'])
+@pytest.mark.parametrize("option3", ['', '.dat', 'dat'])
+@pytest.mark.parametrize("option4", ['', '', os.getcwd()])
+def test_tbft(mapdl, option2, option3, option4):
+    fname = 'expdata0.dat'
+    with open(fname, 'w') as fid:
+        fid.write("""0.819139E-01 0.82788577E+00
+        0.166709E+00 0.15437247E+01
+        0.253960E+00 0.21686152E+01
+        0.343267E+00 0.27201819E+01
+        0.434257E+00 0.32129833E+0""")
```

Comment:
Use the `tmpdir` fixture instead of writing to the local directory.

Reply:
Implementing this.

Reply:
Mmhh... I cannot use `tmpdir` because that will create a file in a tmpdir, hence I would have to modify the @parametrize options, which are outside of that function. It probably can be done, but I feel like it is not worthy.

Also I want to check the option where only fname is given, hence it cannot be in a folder.

Other option is too forget the parametrize and use a more elaborated case.

Reply:
> Other option is too forget the parametrize and use a more elaborated case.

In this case it might be acceptable to create the file in the current working directory given the inflexibility of this command. What I would do instead is `try...finally`. That way we'll still capture all the `AssertionError` raised by our `assert` statements, but always remove the file in `finally`.